brayns/parameters/ParametersManager.cpp

```diff
@@ -50,44 +48,59 @@ unsigned int levenshtein_distance(const std::string& s1, const std::string& s2)
 std::vector<std::string> findSimilarOptions(
     const std::string& name, const std::vector<std::string>& options)
 {
-    size_t bestDist = UINT_MAX;
-    std::vector<std::string> bestOptions;
     constexpr size_t MAX_SUGGESTIONS = 7;
 
-    for (const auto& optionName : options)
+    std::vector<std::string> subStringOptions;
+    std::vector<std::string> levenshteinOptions;
```

Comment:
Why two vectors that need concatenation later on? The original code was fine, I was just suggesting to move the substrings options up.

Reply:
I think it is easier to just keep them separated since then I can clear just the levenshtein vector in the loop.